std/algorithm/iteration.d

```diff
@@ -2868,6 +2871,359 @@ private template ReduceSeedType(E)
     }
 }
 
+/++
+Similar to $(LREF reduce), but returns a range containing the
+successive reduced values. The call $(D scan!(fun)(range, seed))
```

Comment:
AFAIU, there's not much reason to use the old `$(D …)` macro over the new ``` `…` ``` syntax intended on replacing it.